ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -429,23 +438,26 @@ class ( ShowLedgerState (LedgerTables l)
     -> LedgerTables l mk2
     -> f (LedgerTables l mk3)
 
-  zipLedgerTables2 ::
+  -- FIXME: zipLedgerTables2 and zipLedgerTables2A should be renamed to zip2LedgerTables and zip2LedgerTablesA
+  -- Otherwise the number 2 comes too late in the function name.
```

Comment:
Possibly we could add this as a (low-priority?) ticket? I wonder if this will be picked up otherwise

Reply:
I tend to read it as:
- What? "zip"
- With what? "Ledger tables"
- How many? "2"
- Special modifiers? "A"

So I'm not that much in favor of the renaming, but in ultimate instance it doesn't make a difference for me